packages/core/strapi/README.md

```diff
@@ -109,6 +109,8 @@ Strapi only supports maintenance and LTS versions of Node.js. Please refer to th
 | PostgreSQL | 14.0        | 11.0    |
 | SQLite     | 3           | 3       |
 
+Note: For Node 22, the better-sqlite3 package used by your SQLite project should be `>=11.1`
```

Comment:
Is it necessary ? we will only support node 22 at the same time as we upgrade better-sqlite3 🤔 

Sounds like we could say the same thing for a lot of different drivers potentially

Reply:
We don't choose the package version internally, it's part of the project's package.json. So if someone has a project and upgrades to Node 22, they need to be aware of that or else they'll just see that it doesn't work.

Reply:
Ahh, I missed that. The versions we generate projects with already seem to support node 22, but we could look up the minimum supported package versions for node versions as well and add another table in there from now on, could be helpful

Reply:
💯 sounds great